### ISSUE TITLE
FindByAccessibilityId and FindByName failing for WPF despite showing in inspect.exe

### ISSUE THREAD

**Issue by jackarnold** [microsoft/WinAppDriver#1408]
I have the following XAML shown bellow for a TabControl in a WPF application I want to use Appium to test. I've tried using both 
```
var asdf3 = session.FindElementByAccessibilityId("MainTabControl");
```
and 
```
var asdf3 = session.FindElementByName("MainTabControl");
```
Both fail with error 404 in WInAppDriver
```
<TabControl AutomationProperties.AutomationId="MainTabControl" AutomationProperties.Name="MainTabControl" x:Name="MainTabControl" Margin="5,0,5,5" Padding="3" SelectedItem="{Binding SelectedTab}" SelectedIndex="0"
                    SelectionChanged="MainTabControl_SelectionChanged">
```
The Inspect.exe tool shows the following for the control:
![image](https://user-images.githubusercontent.com/18055731/103794257-5e270a80-4ff9-11eb-8b3d-f9055260ddc8.png)

**Comment by liljohnak** [comment]
Try `session.FindElementsByXPath("*//*");` I bet this is a control that Appium doesn't see.

**Comment by Shakevg** [comment]
@jackarnold Also, you can try next https://github.com/microsoft/WinAppDriver/issues/1236#issuecomment-756273289

**Comment by jackarnold** [comment]
I don't remember how this got resolved on my end but the automation testing push never got off the ground. Thanks for all your work building this tool
